crates/sargon/src/gateway_api/models/types/response/transaction/preview/transaction_status.rs

```diff
@@ -0,0 +1,17 @@
+use crate::prelude::*;
+
+#[derive(
+    Deserialize,
+    Serialize,
+    Clone,
+    PartialEq,
+    Eq,
+    Debug,
+    derive_more::Display,
+    uniffi::Enum,
+)]
+pub enum TransactionStatus {
```

Comment:
Can this be replaced by the [TransactionStatus](https://github.com/radixdlt/sargon/blob/main/crates/sargon/src/wrapped_radix_engine_toolkit/high_level/sargon_specific_types/transaction/transaction_status.rs#L15) we have from RET?

Reply:
As I can see they are quite different or am I missing something?

Reply:
I hadn't checked where it was used, that's why I wrongly assumed it was the same. 

It is used under in `TransactionReceipt`, so it isn't actually the `TransactionStatus` but a ` TransactionReceiptStatus`.